mods/ra/maps/exodus/exodus.lua

```diff
@@ -0,0 +1,399 @@
+		Trigger.AfterDelay(ReinforcementsTicks1, function()
+			reinforcements1 = { Reinforcements1[CurrentReinforcement1] }
+			SpawnAlliedUnit(reinforcements1)
+		end)
+	end
+
+	if CurrentReinforcement2 < #Reinforcements2 then
+		CurrentReinforcement2 = CurrentReinforcement2 + 1
+		Trigger.AfterDelay(ReinforcementsTicks2, function()
+			reinforcements2 = { Reinforcements2[CurrentReinforcement2] }
+			SpawnAlliedUnit(reinforcements2)
+		end)
+	end
+end
+
+SovietGroupSize = 5
+SpawnSovietUnits = function()
+	spawnPoint = Utils.Random(SovietEntryPoints)
+	rallyPoint = Utils.Random(SovietRallyPoints)
+	route = { spawnPoint.Location, rallyPoint.Location }
+
+	if DateTime.GameTime >= SovietUnits2Ticks[Difficulty] then
+		units = SovietUnits2
+	else
+		units = SovietUnits1
+	end
+
+	unit = { Utils.Random(units) }
+	Reinforcements.Reinforce(soviets, unit, route)
+
+	delay = math.max(attackAtFrame - 5, minAttackAtFrame)
+
+	Trigger.AfterDelay(delay, SpawnSovietUnits)
+end
+
+SovietParadrop = 0
+SendSovietParadrop = function()
+	local sovietParadrops = SovietParadrops[Difficulty]
+
+	if (SovietParadrop > sovietParadrops) then
+		return
+	end
+
+	SovietParadrop = SovietParadrop + 1
+
+	Utils.Do(humans, function(player)
+		Media.PlaySpeechNotification(player, "SovietForcesApproaching")
+	end)
+
+	local x = Utils.RandomInteger(ParadropBoxTopLeft.Location.X, ParadropBoxBottomRight.Location.X)
+	local y = Utils.RandomInteger(ParadropBoxBottomRight.Location.Y, ParadropBoxTopLeft.Location.Y)
+
+	local randomParadropCell = CPos.New(x, y)
+	local lz = Map.CenterOfCell(randomParadropCell)
+
+	local powerproxy = Actor.Create("powerproxy.paratroopers", false, { Owner = soviets })
+	powerproxy.SendParatroopers(lz)
+	powerproxy.Destroy()
+
+	Trigger.AfterDelay(sovietParadropTicks, SendSovietParadrop)
+end
+
+IdleHunt = function(unit)
+	Trigger.OnIdle(unit, unit.Hunt)
+end
+
+ManageSovietUnits = function()
+	Utils.Do(SovietRallyPoints, function(rallyPoint)
+		local radius = WDist.FromCells(8)
+		local units = Map.ActorsInCircle(rallyPoint.CenterPosition, radius, function(actor)
+			return actor.Owner == soviets and actor.HasProperty("Hunt")
+		end)
+
+		if #units > SovietGroupSize then
+			Utils.Do(units, IdleHunt)
+		end
+	end)
+
+	local scatteredUnits = Utils.Where(soviets.GetGroundAttackers(), function(unit)
+		return not Map.IsNamedActor(unit) and unit.IsIdle and unit.HasProperty("Hunt")
+	end)
+	Utils.Do(scatteredUnits, IdleHunt)
+end
+
+AircraftTargets = function()
+	local targets = Utils.Where(Map.ActorsInWorld, function(a)
+		return (a.Owner == allies1 or a.Owner == allies2)
+			and (a.HasProperty("Move") or a.HasProperty("StartBuildingRepairs"))
+	end)
+
+	return targets
+end
+
+YakAttack = function(yak)
+	local targets = AircraftTargets()
+	if #targets > 0 then
+		target = Utils.Random(targets)
+	else
+		yak.Wait(DateTime.Seconds(5))
+	end
+
+	if yak.AmmoCount() > 0 then
+		yak.Stop()
+		yak.Attack(target)
+	else
+		yak.ReturnToBase()
+		yak.Resupply()
```

Comment:
This shouldn't be needed as pointed out by @obrakmann in https://github.com/OpenRA/OpenRA/pull/11658#discussion_r73580912.